awx/ui_next/src/screens/User/shared/UserForm.jsx

```diff
@@ -0,0 +1,206 @@
+              id="user-username"
+              label={i18n._(t`Username`)}
+              name="username"
+              type="text"
+              validate={required(null, i18n)}
+              isRequired
+            />
+            <FormField
+              id="user-email"
+              label={i18n._(t`Email`)}
+              name="email"
+              validate={requiredEmail(i18n)}
+              isRequired
+            />
+            <PasswordField
+              id="user-password"
+              label={i18n._(t`Password`)}
+              name="password"
+              validate={
+                !user.id
+                  ? required(i18n._(t`This field must not be blank`), i18n)
+                  : () => undefined
+              }
+              isRequired={!user.id}
+            />
+            <PasswordField
+              id="user-confirm-password"
+              label={i18n._(t`Confirm Password`)}
+              name="confirm_password"
+              validate={
+                !user.id
+                  ? required(i18n._(t`This field must not be blank`), i18n)
+                  : () => undefined
+              }
+              isRequired={!user.id}
+            />
+          </FormRow>
+          <FormRow>
+            <FormField
+              id="user-first-name"
+              label={i18n._(t`First Name`)}
+              name="first_name"
+              type="text"
+            />
+            <FormField
+              id="user-last-name"
+              label={i18n._(t`Last Name`)}
+              name="last_name"
+              type="text"
+            />
+            {!user.id && (
+              <Field
+                name="organization"
+                validate={required(
+                  i18n._(t`Select a value for this field`),
+                  i18n
+                )}
+                render={({ form }) => (
+                  <OrganizationLookup
+                    helperTextInvalid={form.errors.organization}
+                    isValid={
+                      !form.touched.organization || !form.errors.organization
+                    }
+                    onBlur={() => form.setFieldTouched('organization')}
+                    onChange={value => {
+                      form.setFieldValue('organization', value.id);
+                      setOrganization(value);
+                    }}
+                    value={organization}
+                    required
+                  />
+                )}
+              />
+            )}
+            <Field
+              name="user_type"
+              render={({ form, field }) => {
+                const isValid =
+                  !form.touched.user_type || !form.errors.user_type;
+                return (
+                  <FormGroup
+                    fieldId="user-type"
+                    helperTextInvalid={form.errors.user_type}
+                    isRequired
+                    isValid={isValid}
+                    label={i18n._(t`Job Type`)}
```

Comment:
I _think_ it should say "User Type" and we probably don't need a tooltip for this field.

Reply:
👍 will remove help text tooltip